glider_kayak_sim/UnderwaterGeoPoint position
geometry_msgs/Quaternion orientation